docs/src/components/Demo.tsx

```diff
@@ -48,7 +48,10 @@ export const Demo = ({
             <TabItem title="Props">
               <View padding={`${tokens.space.medium} 0`}>{propControls}</View>
             </TabItem>
-            {themeControls ?? <TabItem title="Theme">{themeControls}</TabItem>}
```

Comment:
Could we change this to `{themeControls ? <TabItem title="Theme">{themeControls}</TabItem> : null}` ?